core/DataAccess/ArchiveTableDao.php

```diff
@@ -76,7 +76,7 @@ public function getArchiveTableAnalysis($tableDate)
                        COUNT(*) AS count_blob_rows,
                        SUM(OCTET_LENGTH(value)) AS sum_blob_length
                   FROM `$blobTable`
-              GROUP BY idsite, date1, date1, period";
+              GROUP BY idsite, date1, date1, period ORDER BY idsite, period, date1";
```

Comment:
Here it seems we were incorrectly grouping by `date1` twice, plus a suggestion to also order by `date2` for consistency (as above).
```suggestion
              GROUP BY idsite, date1, date2, period ORDER BY idsite, period, date1, date2";
```